.github/workflows/ci-test.yml

```diff
@@ -62,4 +62,4 @@ jobs:
     - name: Run unit tests
       shell: bash
       run: |
-        pytest -m "${{ matrix.which-tests }}"
+        pytest -vv -m "${{ matrix.which-tests }}" tfx
```

Comment:
We had specifically not chosen to put verbose output originally because there often could be an overwhelming amount of output on GitHub and developers can run pytest with verbose output on their local machines. If you would really like verbose output, leave it, by all means.

CC: @peytondmurray

Reply:
Also, is specifying the `tfx` directory strictly necessary here? I believe it is already specified in the [pytest config in `pyproject.toml`](https://github.com/tensorflow/tfx/blob/ee6eaf2c81916170571cc9288112c8a43504c3ef/pyproject.toml#L41)

Reply:
I've reverted this change.